nonebot/adapters/_base.py

```diff
@@ -93,7 +94,7 @@ def register(cls, driver: "Driver", config: "Config"):
     @classmethod
     @abc.abstractmethod
     async def check_permission(cls, driver: "Driver", connection_type: str,
-                               headers: dict, body: Optional[bytes]) -> str:
+                               headers: dict, body: Optional[bytes]) -> Tuple[str, HTTPResponse]:
```

Comment:
response canbe optional
```suggestion
                               headers: dict, body: Optional[bytes]) -> Tuple[str, Optional[HTTPResponse]]:
```